tests/python_client/testcases/test_insert.py

```diff
@@ -1253,6 +1253,21 @@ def test_insert_ids_invalid(self):
         mutation_res, _ = collection_w.insert(
             data=df, check_task=CheckTasks.err_res, check_items=error)
 
+    @pytest.mark.tags(CaseLabel.L2)
+    def test_insert_string_to_int64_pk_field(self):
+        """
+        target: test insert, with using auto id is invalid, which are not int64
+        method: create collection and insert entities in it
+        expected: raise exception
+        """
+        collection_name = cf.gen_unique_str(prefix)
+        collection_w = self.init_collection_wrap(name=collection_name)
+        df = cf.gen_default_dataframe_data(1)
+        df[ct.default_int64_field_name] = ["2000000"]
```

Comment:
I'd suggest gen 100 entities, and update one of them to be a string.

Reply:
Good advice, I have already changed it.